tests/integration-test/bpf/app-test.bpf.c

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause)
+// Copyright Authors of bpfman
+
+// clang-format off
+#include <linux/bpf.h>
+#include <linux/pkt_cls.h>
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_tracing.h>
+// clang-format on
+
+volatile const __u8 GLOBAL_u8 = 0;
+volatile const __u32 GLOBAL_u32 = 0;
+
+void print_globals(char *prefix) {
+  bpf_printk("%s: GLOBAL_u8: 0x%02X, GLOBAL_u32: 0x%08X", prefix, GLOBAL_u8,
+             GLOBAL_u32);
+}
+
+SEC("fentry/do_unlinkat")
+int BPF_PROG(fentry_test) {
+  print_globals("FEN");
+  return 0;
+}
+
+SEC("fexit/do_unlinkat")
+int BPF_PROG(fexit_test) {
+  print_globals("FEX");
+  return 0;
+}
+
+SEC("kprobe/kprobe_test")
+int kprobe_test(struct pt_regs *ctx) {
+  print_globals(" KP");
+  return 0;
+}
+
+SEC("kretprobe/kretprobe_test")
+int kretprobe_test(struct pt_regs *ctx) {
+  print_globals("KRP");
+  return 0;
+}
+
+SEC("classifier/tc_pass_test")
+int tc_pass_test(struct __sk_buff *skb) {
+  print_globals(" TC");
+  return TC_ACT_OK;
+}
+
+SEC("classifier/tcx_pass_test")
```

Comment:
can we have tcx_ingress and tcx_egress

Reply:
I'm not sure what you mean.  These functions can be attached in either the ingress or egress directions or both.

Reply:
since u were generating one BC for all different hook and I care about bidirection TC and TCX hooks hence was suggesting adding them for completeness

Reply:
I created one for each action.  They can be attached to any hook point in either direction, so I was just keeping them generic.